ui/src/app/pages/data/data-page.tsx

```diff
@@ -107,38 +105,24 @@ export const DataPage = withCurrentWorkspace()(class extends React.Component<
 
   async loadResources() {
     try {
-      const {namespace, id, accessLevel} = this.props.workspace;
+      const {namespace, id} = this.props.workspace;
 
       this.setState({
         isLoading: true
       });
-      const [cohorts, cohortReviews, conceptSets, dataSets] = await Promise.all([
-        fetchWithGlobalErrorHandler(() => cohortsApi().getCohortsInWorkspace(namespace, id)),
-        cohortReviewApi().getCohortReviewsInWorkspace(namespace, id),
-        conceptSetsApi().getConceptSetsInWorkspace(namespace, id),
-        dataSetApi().getDataSetsInWorkspace(namespace, id)
-      ]);
-      // Show all concept set except the Dummy demographics Concept set created to be used only
-      // in dataset
-      conceptSets.items = conceptSets.items
-          .filter(conceptSet => conceptSet.domain !== Domain.PERSON);
-      this.setState({
-        existingCohortName: cohorts.items.map(cohort => cohort.name),
-        existingCohortReviewName: cohortReviews.items.map(review => review.cohortName),
-        existingConceptSetName: conceptSets.items.map(conceptSet => conceptSet.name),
-        existingDataSetName: dataSets.items.map(dataSet => dataSet.name)
-      });
-      let list: RecentResource[] = [];
-      list = list.concat(convertToResources(cohorts.items, namespace,
-        id, accessLevel as unknown as WorkspaceAccessLevel, ResourceType.COHORT));
-      list = list.concat(convertToResources(cohortReviews.items, namespace,
-        id, accessLevel as unknown as WorkspaceAccessLevel, ResourceType.COHORTREVIEW));
-      list = list.concat(convertToResources(conceptSets.items, namespace,
-        id, accessLevel as unknown as WorkspaceAccessLevel, ResourceType.CONCEPTSET));
-      list = list.concat(convertToResources(dataSets.items, namespace,
-        id, accessLevel as unknown as WorkspaceAccessLevel, ResourceType.DATASET));
+      const results = await workspacesApi().getCdrSelectors(namespace, id);
+
+      results.map(result => result.modifiedTime ? result.modifiedTime = new Date(Number(result.modifiedTime)).toDateString() : null);
```

Comment:
if there's a ticket yet for how this may be changed, please put it in a comment.